packages/react-ui-core/src/Icon/Icon.js

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from 'react'
+import RPT from 'prop-types'
+
+export default class Icon extends Component {
```

Comment:
That may be true, but that may not be the case on rent-js or any of our other sites.